### PR TITLE
[helm/charts] Add configmap

### PR DESCRIPTION
**Pull request by faizanahmad055**
Signed-off-by: Faizan Ahmad
username: faizanahmad055
email: faizan.ahmad55@outlook.com

#### What this PR does / why we need it:
Configmaps chart is used in conjunction with other charts. It can come in handy when a public chart does not provide support for the additional configmaps. In that scenario, this chart can be used to deploy additional configmaps.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by faizanahmad055** [comment]
Hi @seanknox and @mgoodness

Can you guys please take a look

File: stable/configmaps/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# configmaps
+
+This chart is used to apply configmaps. You can specify any number of configmaps. This chart can come in handy when a public chart does not offer additional configmaps. You have to use the following format to specify a configmap:
+
+```bash
```
Comment (philoserf) [inline stable/configmaps/README.md]: This is YAML not shell.

File: stable/configmaps/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+configmaps chart is used with conjunction of other charts. It can come in handy when a public chart does not provide support for additional configmaps. In that scenario, this chart can be used to deploy additional configmaps.
```
Comment (philoserf) [inline stable/configmaps/templates/NOTES.txt]: NOTES.txt is intended as a post install message. This is simply a restatement of the purpose from the README.

File: stable/configmaps/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# configmaps
+
+This chart is used to apply configmaps. You can specify any number of configmaps. This chart can come in handy when a public chart does not offer additional configmaps. You have to use the following format to specify a configmap:
```
Comment (philoserf) [inline stable/configmaps/README.md:3]: I think the README need a little more information about the use cases this chart solves. Perhaps a specific example of using it with one of the “public charts”.

**Review by philoserf — CHANGES_REQUESTED** [review]

File: stable/configmaps/README.md (hunk 3 above)
Comment (faizanahmad055) [reply stable/configmaps/README.md:3]: Updated

Comment (faizanahmad055) [reply stable/configmaps/README.md:3]: Updated

File: stable/configmaps/templates/NOTES.txt (hunk 2 above)
Comment (faizanahmad055) [reply stable/configmaps/templates/NOTES.txt]: Updated

**Comment by philoserf** [comment]
I can’t move the pull request along @faizanahmad055. Like you, I am a member of the community. I’m not an approver.

**Comment by faizanahmad055** [comment]
Hi @foxish and @mgoodness
Ping

**Comment by faizanahmad055** [comment]
Hi @foxish and @mgoodness
Ping

**Comment by faizanahmad055** [comment]
Hi @seanknox and @mgoodness
Can you guys take a look? All requested changes have been implemented. If there are any more; please do let me know.

**Comment by faizanahmad055** [comment]
Hi @seanknox and @mgoodness
Ping

**Comment by faizanahmad055** [comment]
Hi @seanknox and @mgoodness
Any update on this please

**Pull request closed without merging**